launcher/verifier/fake/fakeverifier.go

```diff
@@ -0,0 +1,60 @@
+// Package fake is a fake implementation of the Client interface for testing.
+package fake
+
+import (
+	"context"
+	"crypto"
+
+	"github.com/golang-jwt/jwt/v4"
+	"github.com/google/go-tpm-tools/launcher/verifier"
+)
+
+type fakeClient struct {
+	//pbClient servpb.AttestationVerifierClient
+	signer crypto.Signer
+}
+
+// NewClient contructs a new fake client given a crypto.Signer.
+func NewClient(signer crypto.Signer) verifier.Client {
+	return &fakeClient{signer}
+}
+
+// CreateChallenge returns a hard coded, basic challenge.
+//
+// If you have found this method is insufficient for your tests, this class must be updated to
+// allow for better testing.
+func (fc *fakeClient) CreateChallenge(ctx context.Context) (*verifier.Challenge, error) {
+	return &verifier.Challenge{
+		Name:  "FakeName",
+		Nonce: []byte{0x0},
+	}, nil
+}
+
+// VerifyAttestation does basic checks and returns a hard coded attestation response.
+//
+// If you have found this method is insufficient for your tests, this class must be updated to
+// allow for better testing.
+func (fc *fakeClient) VerifyAttestation(ctx context.Context, request verifier.VerifyAttestationRequest) (*verifier.VerifyAttestationResponse, error) {
+	// Determine signing algorithm.
+	signingMethod := jwt.SigningMethodRS256
+	now := jwt.TimeFunc()
+	claims := jwt.RegisteredClaims{
+		IssuedAt:  &jwt.NumericDate{Time: now},
+		NotBefore: &jwt.NumericDate{Time: now},
+		ExpiresAt: &jwt.NumericDate{Time: now.Add(60 * 60 * 1e9)}, // Add takes nanoseconds
+		Audience:  []string{"TestingAudience"},
+		Issuer:    "TestingIssuer",
+		Subject:   "TestingSubject",
```

Comment:
Can we have these fields also contain real-ish data?

- `Audiance:` `"https://sts.googleapis.com/"`
- `Issuer:` `"https://confidentialcomputing.googleapis.com/"`
- `Subject:` `"https://www.googleapis.com/compute/v1/projects/fakeProject/zones/fakeZone/instances/fakeInstance"`